src/test/java/uk/gov/companieshouse/filetransferservice/service/impl/AmazonFileTransferImplTest.java

```diff
@@ -25,17 +36,8 @@
 import java.util.Map;
 import java.util.Optional;
 
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.atLeastOnce;
-import static org.mockito.Mockito.doNothing;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-
 class AmazonFileTransferImplTest {
+    private static final String TEST_FILE_NAME = "file.pdf";
```

Comment:
Where is this file located? Is it in resources? Is it possible to include a .xhtml file instead of a pdf for testing purposes?

Reply:
It's not an actual file. It is just a value used to create mocks. It's a constant so it can be used to create a mock and verify the call with the same value. 

It could be changed to an XHTML extension, but this would offer no benefit as it's just a string comparision of the names. There is no content being checked.